go.mod

```diff
@@ -2,28 +2,24 @@ module github.com/bom-squad/protobom
 
 go 1.20
 
-require google.golang.org/protobuf v1.31.0
-
 require (
 	github.com/CycloneDX/cyclonedx-go v0.7.1
 	github.com/google/uuid v1.3.0
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/cobra v1.7.0
 	github.com/stretchr/testify v1.8.4
+	google.golang.org/protobuf v1.31.0
```

Comment:
The PR seems to be introducing lots of small white space and other minor changes. It is reshuffling `go.mod` here. Since it got into a merge conflict from the last dependabot update and it is not really related to PR's purpose I wonder if we should just drop this change from the commit? It's going to merge conflict #53 (or this one again) as it is.

Reply:
Yeah, I believe we can, I rebuilt go.mod since it got the common "problem" of multiple require () blocks. 

Also, I know it added a bunch of whitespaces, that's because I added a yaml linter. I will revert some of those and delay that into a later commit

Reply:
Ah gotcha. OK, I pulled the require blocks together in go.mod, give it a rebase and you should see them joined now :+1: